hera_pspec/uvpspec.py

```diff
@@ -1341,14 +1349,14 @@ def read_from_group(self, grp, just_meta=False, spws=None, bls=None,
         # Clear all data in the current object
         self._clear()
 
-        # Load-in meta data
+        # Load-in meta data including deprecated. 
         for k in grp.attrs:
-            if k in self._meta_attrs:
+            if k in self._meta_attrs or k in self._meta_deprecated:
                 val = grp.attrs[k]
                 if isinstance(val, bytes): val = val.decode() # bytes -> str
                 setattr(self, k, val)
         for k in grp:
-            if k in self._meta_dsets:
+            if k in self._meta_dsets or k in self._meta_dsets_deprecated:
                 setattr(self, k, grp[k][:])
 
         # Backwards compatibility: pol_array exists (not polpair_array)
```

Comment:
Not covered by tests according to codecov

Reply:
The reason this line is never covered is because there are not yet any items in `_meta_dsets_deprecated`. I created this as a helpful placeholder to deal with any future changes to `uvpspec.meta_dsets` that might occur. As of now, there is no concrete scenario where this line of code will actually get called. If you think this is unecessary we can drop it.